build-system/tasks/validate-html-fixtures.js

```diff
@@ -113,7 +113,13 @@ async function runCheck(filesToCheck) {
     }
   }
   if (foundValidationErrors) {
-    throw new Error('Please address the errors listed above.');
+    throw new Error(
+      'Please address the errors listed above.\n' +
+        `  - 💡 If a failing file is a ${cyan('Bento')} document,` +
+        `it's not meant to be valid AMP.\n` +
+        `    Place it under any directory named "${cyan('bento')}",` +
+        `like ${cyan('examples/bento/')}, so that it's not validated.`
+    );
```

Comment:
To make this message more readable, I suggest logging it after line 111 instead of mixing it with the error message that's printed along with the call-stack. Here's a proposed replacement for line 111 that follows our logging convention.

```js
      log(red('ERROR:'), 'Found errors in', cyan(format), 'fixtures.');
      log(
        '⤷ If a failing file is a',
        cyan('Bento'),
        'document, it is not meant to be valid AMP.'
      );
      log(
        '⤷ Place it under any directory named',
        cyan('bento'),
        'like',
        cyan('examples/bento/'),
        'so that it is not validated.'
      );
```